user-management-service/app/src/main/java/com/microservices/usermanagement/dto/UserDto.java

```diff
@@ -0,0 +1,24 @@
+package com.microservices.usermanagement.dto;
+
+import com.microservices.usermanagement.entity.CardInfo;
+import com.microservices.usermanagement.entity.UserEntity;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class UserDto {
+    private int id;
+
+    private String login;
+
+    private CardInfo cardInfo;
+
+    public UserDto(UserEntity userEntity) {
```

Comment:
Add lombok NonNull annotation to parameter

Reply:
added